ACKNOWLEDGMENTS.md

```diff
@@ -7,6 +7,7 @@ with a short description of your contribution(s) below. For example:
 
 MLX was developed with contributions from the following individuals:
 
+- Nripesh Niketan: Added `dropout3d` ops.
 - Nripesh Niketan: Added `softsign`, `softmax`, `hardswish`, `logsoftmax` activation functions.
```

Comment:
I think it's simpler if you just append to your existing line rather than make a new line here (that's also what other have been doing!)

Reply:
Done!